flink-table/flink-sql-client/src/test/resources/sql/set.q

```diff
@@ -313,3 +303,11 @@ SELECT id, func1(str) FROM (VALUES (1, 'Hello World')) AS T(id, str) ;
 +----+-------------+--------------------------------+
 Received a total of 1 row
 !ok
+
+REMOVE JAR '$VAR_UDF_JAR_PATH';
```

Comment:
ditto

Reply:
The show jars statement before the remove jar is at line line 281